shell/common/shell.cc

```diff
@@ -1328,6 +1328,18 @@ void Shell::OnEngineHandlePlatformMessage(
   }
 }
 
+void Shell::OnEngineChannelUpdate(std::string name, bool listening) {
+  FML_DCHECK(is_set_up_);
+  FML_DCHECK(task_runners_.GetUITaskRunner()->RunsTasksOnCurrentThread());
+
+  task_runners_.GetPlatformTaskRunner()->PostTask(
+      [view = platform_view_->GetWeakPtr(), name = name, listening] {
```

Comment:
Can we move this too to avoid a copy?

```suggestion
      [view = platform_view_->GetWeakPtr(), name = std::move(name), listening] {
```

Reply:
Putting `std::move` there produced an error for an unnecessary `std::move` in one of the postsub tests when I tried it.

Reply:
Oh ok. I don't understand C++ 😅